Packs/GoogleCloudLogging/Integrations/GoogleCloudLogging/README.md

```diff
@@ -0,0 +1,213 @@
+With Google Cloud Logging, users can centralize all their logs in a single location, making it easier to troubleshoot issues and gain insights from their data.
+
+## Configure Google Cloud Logging on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Google Cloud Logging.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Service Account JSON | User's Service Account key in JSON format. | True |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### gcp-logging-log-entries-list
```

Comment:
Let's add here also the necessary permissions for the command (scopes, IAM permissions etc)
under permission section